Week3/HW3/3_moreJavascript/10_vehicleAd.js

```diff
@@ -0,0 +1,12 @@
+let vehicle = ["tram","metro", "motorbike", "caravan", "bike"];
+
+function advert(arr) { 
+    txt = "Amazing Joe's Garage, we service "; 
+    for ( i = 0 ; i < arr.length -2; i++) {
+        txt += arr[i]+"s, " 
+    }; 
+    return txt += arr[arr.length -2 ]+"s" + " and " + arr[arr.length -1]+"s"; 
+}
+
+advert(vehicle);
+// prints: "Amazing Joe's Garage, we service trams, metros, motorbikes, caravans and bikes"
```

Comment:
👍 for logic but a couple things to take care of next time onwards:
1. Using `const` instead of `let` while creating `vehicle` object since we aren't reassigning it anywhere else.
2. You didn't use any keyword while creating variables `txt` or `i` inside the function. This means JavaScript created both these variables in global scope which is bad. 😱 
3. While choosing function names, it is a good practice to pick function names starting with a verb. For example, something like `printAdvertisement` or `getAdvertisement` would be a suitable name for this function. Have a look here again please when it comes to naming variables - https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md

Reply:
Thank you for approving my work.
Alright, I will remember them and go through the site.